webpack.config.js

```diff
@@ -0,0 +1,32 @@
+var path = require('path');
+
+module.exports = {
+  entry: __dirname + '/src/main.jsx',
+  output: {
+    path: __dirname + '/build/js',
+    filename: '[name].bundle.js'
+  },
+  externals: {
+    'react': 'React',
```

Comment:
@toolness Not quite the same as setting the `NODE_ENV=production` (which we are doing in the staging and production environments). `webpack -p` is a shorthand for `webpack --optimize-minimize --optimize-occurence-order`